packages/dds/tree/src/simple-tree/arrayNodeTypes.ts

```diff
@@ -0,0 +1,83 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import type { TreeArrayNode } from "./arrayNode.js";
+import type {
+	ImplicitAllowedTypes,
+	InsertableTreeNodeFromImplicitAllowedTypes,
+} from "./schemaTypes.js";
+import {
+	NodeKind,
+	type TreeNodeSchemaClass,
+	type TreeNodeSchema,
+	type TreeNodeSchemaNonClass,
+	type WithType,
+} from "./core/index.js";
+
+import type { SimpleArrayNodeSchema } from "./simpleSchema.js";
+
+/**
+ * A schema for customizable {@link (TreeArrayNode:interface)}s.
+ * @sealed
+ * @alpha
+ */
+export interface ArrayNodeCustomizableSchema<
```

Comment:
"Customizable" and "PojoEmulation" are not terms I remember seeing before. I'm assuming "Customizable" corresponds to class-based schema, and "PojoEmulation" corresponds to structurally named schema? Sanity checking my understanding, and also making sure we're aligning with existing terminology.

Reply:
"Customizable" and "Pojo" are indeed the official (and already documented) terms, though we still use "class-based" and "structural" colloquially.

Reply:
They are in the API for SchemaFactory https://fluidframework.com/docs/api/fluid-framework/schemafactory-class#schemafactory-remarks